service-configure.js

```diff
@@ -179,6 +180,15 @@
         var questions = [
             {
                 type: 'input',
+                name: 'testfairyConnectEndpoint',
+                message: 'What is your TestFairy Connect endpoint (do not change unless you have custom testfairy domain)?',
```

Comment:
All developers now have a custom domain (you wrote that code ;)). Can you please change to "What is your TestFairy Connect endpoint (eg, https://acme.testfairy.com/connect)?